.github/actions/spell-check/expect.txt

```diff
@@ -140,12 +140,14 @@ backend
 backtracer
 bak
 bbwe
+bca
 bcc
 bck
 Bcl
 BEGINLABELEDIT
 betadele
 betsegaw
+bfee
```

Comment:
I am sure this is for the guid. Bit I am not s fan of it. Couldn't we great a regex rule for spellchecker?

Reply:
This seems to be a generic problem... IMHO, should be solved outside of this PR.

Reply:
Sure. Nut we should think about in general.